helm/values.yaml

```diff
@@ -173,6 +173,12 @@ guacamole:
 # Never use 'local' in production!
 target: other
 
+# Activate the development mode.
+# NEVER use this in production, it will make your application unsafe.
+# The development mode changes the following options:
+# - Initialize Papyrus and use local images per default (instead of the public Github images)
+development: false
```

Comment:
It's a bit like `target: local`. Shouldn't we use that instead? Or drop `target`?

Reply:
There is a slight difference. People started using it with target `local` on local servers. However, it's not a development server, it is used for "production". The name local is therefore not clear enough. 

The goal is to drop target and just have boolean value for the development mode instead of a enum.